qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -128,6 +131,15 @@
             Windows:
                 # Windows guest only works with qxl driver.
                 vga = "qxl"
+        - check_guest_file:
+            gagent_check_type = guest_file
+            file_path = "/tmp/"
+            cmd_read = "cat"
+            cmd_change_black = "sed -i 's/guest-file-[a-zA-Z]*,//g' /etc/sysconfig/qemu-ga"
+            cmd_verify = "grep -nr 'guest-file' /etc/sysconfig/qemu-ga"
```

Comment:
@xiagao should consider to use "grep -nr 'BLACKLIST_RPC=*guest-file*' /etc/sysconfig/qemu-ga", avoid get useless line with guest-file key word. If so, it is more accurate.